### PR TITLE
Blog week 9 2020

### PR DESCRIPTION
**Pull request by sakalosj**

### CODE REVIEW

File: content/posts/February-2020.md [hunk 1]
```diff
@@ -84,3 +84,28 @@ Following bugs were fixed:
...
+
+## Week 9
+
+Dist-git patches are applied with `-p1` [(#730)]. Packit specfile code was improved [(#736)], [(#734)].
+Fedora-32 was added to 'fedora-all' and 'fedora-development' aliases [(#731)].
+Documentation was improved by adding links to user configuration [(#726)].
+
+Packit-service was adjusted, so we don't need metadata for copr build [(#431)], ansible deployment playbooks
+were adjusted, to use same playbooks in zuul and localy [(#429)], handlers were moved to separate module[(#426)]. 
+Build-helper classes were refactored [(#424)].
```
Comment (lachmanfrantisek) [inline content/posts/February-2020.md]: We can maybe merge the sentences about refactoring.

File: content/posts/February-2020.md [hunk 2]
```diff
@@ -84,3 +84,28 @@ Following bugs were fixed:
 [(#715)]: https://github.com/packit-service/packit/pull/715
 [(#406)]: https://github.com/packit-service/packit-service/pull/406
 [(#420)]: https://github.com/packit-service/packit-service/pull/420
+
+## Week 9
+
+Dist-git patches are applied with `-p1` [(#730)]. Packit specfile code was improved [(#736)], [(#734)].
```
Comment (lachmanfrantisek) [inline content/posts/February-2020.md]:
```suggestion
Dist-git patches are newly applied with `-p1` [(#730)]. Packit specfile code was improved [(#736)], [(#734)].
```

File: content/posts/February-2020.md [hunk 3]
```diff
@@ -84,3 +84,28 @@ Following bugs were fixed:
...
+Fedora-32 was added to 'fedora-all' and 'fedora-development' aliases [(#731)].
+Documentation was improved by adding links to user configuration [(#726)].
+
+Packit-service was adjusted, so we don't need metadata for copr build [(#431)], ansible deployment playbooks
+were adjusted, to use same playbooks in zuul and localy [(#429)], handlers were moved to separate module[(#426)]. 
+Build-helper classes were refactored [(#424)].
+
+Bug related to incorrect pull-request filtering fixed [(#441)]
```
Comment (lachmanfrantisek) [inline content/posts/February-2020.md]:
```suggestion
Bug related to incorrect pull-request filtering fixed [(#441)].
```

Comment (jpopelka) [inline content/posts/February-2020.md]:
```suggestion
Bug related to incorrect pull-request filtering was fixed [(#441)].
```

Comment (sakalosj) [reply content/posts/February-2020.md]: fixed

File: content/posts/February-2020.md (hunk 2 above)
Comment (sakalosj) [reply content/posts/February-2020.md]: fixed

File: content/posts/February-2020.md (hunk 1 above)
Comment (sakalosj) [reply content/posts/February-2020.md]: fixed

File: content/posts/February-2020.md [hunk 4]
```diff
@@ -84,3 +84,28 @@ Following bugs were fixed:
...
 [(#420)]: https://github.com/packit-service/packit-service/pull/420
+
+## Week 9
+
+Dist-git patches are newly applied with `-p1` [(#730)]. Packit specfile code was improved [(#736)], [(#734)].
+Fedora-32 was added to 'fedora-all' and 'fedora-development' aliases [(#731)].
+Documentation was improved by adding links to user configuration [(#726)].
+
+Packit-service was adjusted, so we don't need metadata for copr build [(#431)], ansible deployment playbooks
+were adjusted, to use same playbooks in zuul and localy [(#429)].
```
Comment (jpopelka) [inline content/posts/February-2020.md]:
```suggestion
Packit-service was adjusted, so we don't need metadata for copr build [(#431)].
Ansible playbooks in deployment repo were adjusted to use the same playbooks in Zuul and locally [(#429)].
```

Comment (sakalosj) [reply content/posts/February-2020.md]: fixed

File: content/posts/February-2020.md (hunk 3 above)
Comment (sakalosj) [reply content/posts/February-2020.md]: fixed

File: content/posts/February-2020.md [hunk 5]
```diff
@@ -84,3 +84,28 @@ Following bugs were fixed:
...
+[(#736)]: https://github.com/packit-service/packit/pull/736
+[(#734)]: https://github.com/packit-service/packit/pull/734
+[(#731)]: https://github.com/packit-service/packit/pull/731
+[(#726)]: https://github.com/packit-service/packit/pull/726
+[(#431)]: https://github.com/packit-service/packit-service/pull/431
+[(#429)]: https://github.com/packit-service/packit-service/pull/429
+[(#426)]: https://github.com/packit-service/packit-service/pull/426
+[(#424)]: https://github.com/packit-service/packit-service/pull/424
+[(#441)]: https://github.com/packit-service/packit-service/pull/441
+
```
Comment (TomasTomecek) [inline content/posts/February-2020.md]: please remove these

**Comment by TomasTomecek** [comment]
I'm going to cherry pick your commit, do the edits myself, and then provide a new entry for this wekk.

**Pull request closed without merging**